.travis.yml

```diff
@@ -1,21 +1,13 @@
 language: php
 
 php:
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
```

Comment:
can we keep 7.1 for compat with SF 4.4 lowest version?

Reply:
I don't think so :disappointed: 

The `matthiasnoback/symfony-dependency-injection-test` 4.x branches conflict with PHPUnit <8.0, which only supports PHP 7.2+

Reply:
that's solvable by using the phpunit-bridge actually